dropshots/src/androidTest/kotlin/com/dropbox/dropshots/DropshotsTest.kt

```diff
@@ -86,20 +86,29 @@ class DropshotsTest {
 
   @Test
   fun writesOutputImageOnFailure() {
+    var failed = false
```

Comment:
This is the pattern I usually follow for cases in which you can't easily `fail()` because you're trying to catch an `AssertionError`.  I think since it's confined to the test we're okay, but I'm happy to rename if you think it important.